src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java

```diff
@@ -5,6 +5,10 @@
 
 import com.paneedah.mwc.ModernWarfareMod;
 import com.paneedah.mwc.models.*;
+import com.paneedah.mwc.models.AACHoneyBadgerSilencer;
+import com.paneedah.mwc.models.Acog2;
+import com.paneedah.mwc.models.AcogReticle;
+import com.paneedah.mwc.models.AcogScope2;
```

Comment:
Again what's up with the imports

Reply:
IDK It just happened the second I moved a file by accident